.github/workflows/test-gql-mutations.yml

```
@@ -0,0 +1,55 @@
+# Copyright 2022 Democratized Data Foundation
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+name: Test GQL Mutations
```

todo: All workflows should end with the `Workflow` ending.

```suggestion
name: Test GQL Mutations Workflow
```

---

Ugh, I thought I matched all the naming conventions I could spot, will change.

- [ ] Rename workflow (see other comment too)